### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by IMACULGY**
Added an image, a "Contribution Section" and headings for sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,60 @@ Equipment
...
 This module adds a basic equipment system, a character screen, and a few pieces of example equipment.
 
-Credits for images:
+![Alt text](https://i.imgur.com/YX4JC0H.png)
+
+# Contribution
+Fork and clone this repository locally.
+
+## Creating an Equipment
+To create an Equipment, create a prefab similar to the ones [here](https://github.com/Terasology/Equipment/tree/master/assets/prefabs/Equipment) at `Equipment/assets/prefabs/Equipment/` and add the associated icon to `Equipment/assets/textures/`. For instance, to create an Equipment called "DiamondAxe," the prefab should look like:
```
Comment (vampcat) [inline README.md]: I've mostly heard `in a folder`, not `at a folder`. Can we change the text to read
to the ones here "in" `Equirement/...` ?

**Review by vampcat — CHANGES_REQUESTED** [review]
Just one tiny change. Apart from that, all looks good :)

**Comment by IMACULGY** [comment]
Sure, changed "at" to "in" when necessary. Sorry about that!

**Review by llvieira — APPROVED** [review]

**Comment by pollend** [comment]
one more small change. can you change the contribution similar to this:

image

https://github.com/Terasology/Potions

**Comment by IMACULGY** [comment]
Done, thanks.

**Review by pollend — APPROVED** [review]

**Review by pollend — APPROVED** [review]

**Comment by casals** [comment]
Closing since the README alterations are outdated. Moving the relevant information to the wiki.

**Pull request closed without merging**